src/lib/button/button.ts

```diff
@@ -110,6 +110,8 @@ export class MdButton {
     return el.hasAttribute('md-' + suffix) || el.hasAttribute('mat-' + suffix);
   });
 
+  _isIconButton: boolean = this._getHostElement().hasAttribute('md-icon-button');
```

Comment:
How would it? The method `_isRoundButton` checks both prefixes? (See [here](https://github.com/angular/material2/pull/3208/files#diff-96337821f283af0f38c4fd8ed4c98065R110))